Blackjack/Rules.cs

```diff
@@ -21,5 +25,25 @@ public static Choice ShouldDealerHitAgain(int score)
         {
             return score < 17 ? Choice.Hit : Choice.Stay;
         }
+
+        public static string GoneBlackjack(int playerScore, int dealerScore) // TODO: ternary operator
+        {
+            if (IsBlackjack(playerScore) && IsBlackjack(dealerScore)) return Messages.Tie;
+            if (IsBlackjack(playerScore)) return Messages.PlayerWins;
+            return Messages.DealerWins;

```

Comment:
If neither the player nor the dealer has a score of 21, this function will return the dealer wins message. How it evaluates:
condition 1: if both player and dealer has blackjack 
true => it's a tie
false => go to next line (condition 2)
condition 2: if the player has blackjack 
true => player wins 
false => go to next line
next line of code: return dealer wins message